packages/buidler-etherscan/src/config.ts

```diff
@@ -2,10 +2,13 @@ import { ResolvedBuidlerConfig } from "@nomiclabs/buidler/types";
 
 import { EtherscanConfig } from "./types";
 
+// TODO: This function doesn't guarantee an EtherscanConfig object.
+// Some of our users don't use TypeScript and instead use plain JavaScript.
+// This means that the config.etherscan field could actually be anything.
 export function getDefaultEtherscanConfig(
   config: ResolvedBuidlerConfig
 ): EtherscanConfig {
-  const defaultConfig = { url: "https://api.etherscan.io/api", apiKey: "" };
+  const defaultConfig = { apiKey: "" };
```

Comment:
There's now a better way to handle default config values, which you can see in action here: https://github.com/nomiclabs/buidler/blob/0183504555c2aed2e3ea7e63b6ddeb656ce7d4f7/packages/buidler-ganache/src/index.ts#L29